src/lib/pages/account-details/index.tsx

```diff
@@ -344,7 +344,7 @@ const AccountDetails = () => {
   // TODO: change to `Addr` for correctness (i.e. interchain account)
   const accountAddressParam = getFirstQueryParam(
```

Comment:
I think we should lowerCase inside `validateAddress`

Reply:
![Screen Shot 2566-06-13 at 15 03 50](https://github.com/alleslabs/celatone-frontend/assets/52099595/c320fc38-bc43-4218-a8d7-6ba6fb1fb6d8)
if we. dont lower case here this area will be all uppercase, is that okay?

Reply:
I see